gradio/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "gradio",
-	"version": "4.29.0",
+	"version": "4.30.0",
```

Comment:
I assume this was for testing purposes, let's revert?

Reply:
Yea I needed this for the tests to test the new logic. I asked @pngwn and he thinks its ok cause the release pr will override this anyways. But I can revert if we don't mind not having the test coverage until we release!

Reply:
Ah got it, yea we can leave it